src/components/AddRepoForm/__tests__/AddRepoForm.spec.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import { Provider } from 'react-redux'
+import configureStore from 'redux-mock-store'
+import { mount } from 'enzyme'
+import AddRepoForm from '../AddRepoForm'
+import { getRepository } from '../../../actions'
+
+const mockStore = configureStore([])
+ 
+describe('AddRepoForm', () => {
+  let store;
+ 
+  beforeEach(() => {
+    store = mockStore({
+      repositories: {
+        data: {},
+        error: "Repository not found"
+      }
+    })
+
+    store.dispatch = jest.fn()
+  })
+ 
+  it('should render correctly', () => {
+    const component = mount(
```

Comment:
We don't use mount for snapshots, only shallow to avoid mocking components on the next levels